packages/core/src/common/metrics.ts

```diff
@@ -20,24 +20,24 @@ export class MetricsService {
 
   ponder_indexing_total_seconds: prometheus.Gauge;
   ponder_indexing_completed_seconds: prometheus.Gauge;
-  ponder_indexing_completed_events: prometheus.Gauge<"network" | "event">;
+  ponder_indexing_completed_events: prometheus.Gauge<"event">;
```

Comment:
We'll need a changeset for the metric label changes - could update the one we already have on `0.6`